modules/exploits/linux/local/gameoverlay_privesc.rb

```diff
@@ -0,0 +1,143 @@
+class MetasploitModule < Msf::Exploit::Local
+
+  prepend Msf::Exploit::Remote::AutoCheck
+  include Msf::Post::Linux::System
+  include Msf::Post::Linux::Kernel
+  include Msf::Post::File
+  include Msf::Exploit::FileDropper
+
+  def initialize(info = {})
+    super(
+      update_info(
+        info,
+        'Name' => 'GameOver(lay) Privilege Escalation and Container Escape',
+        'Description' => %q{
+          This module exploits the use of unsafe functions in a number of Ubuntu kernels
+          utilizing vunerable versions of overlayfs. To mitigate CVE-2021-3493 the Linux
```

Comment:
I'm guessing this reference to cve-2021-3493 is a copy/paste remnant from the older overlayfs module?

Reply:
No, this is not a copy and paste from the other module. The description I gave for this module may have been too high level. CVE-2021-3493 effects the Linux kernel and then it was patched. Ubuntu made separate changes to the kernel and those changes did not incorporate the patches made to the original Linux kernel resulting in [CVE-2023-32629](https://ubuntu.com/security/CVE-2023-32629) and [CVE-2023-2640](https://ubuntu.com/security/CVE-2023-2640). 

I'm not sure if you've read [this article already](https://www.wiz.io/blog/ubuntu-overlayfs-vulnerability) but it does a tremendous job explaining the technical workings of the vulnerabilities within OverlayFS and how they arose. Should we just take the mention of CVE-2021-3493 out of the description for simplicity sake?